CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ include(${CCF_DIR}/cmake/preproject.cmake)
 
 project(
   ccf
-  VERSION 0.8.2
+  VERSION 0.8.2 # CCF_VERSION < This is a search marker for checks.yml
```

Comment:
for `check-versions.sh`?

Reply:
Updated comment, removed reference to file. This is just a "don't delete this line" comment, and if we want to find where its used we search for "CCF_VERSION".